src/redux/actions/ClustersActions.js

```diff
@@ -10,7 +10,7 @@ export const startFetchingClusters = () => ({
 
 export const getClustersSuccess = (response) => ({
   type: GET_CLUSTERS,
-  payload: response.data.data,
+  payload: response.data.data.clusters,
```

Comment:
And you by changing this line you caused Errors on the `Admin side`, They depend on this object and now app just crashes, Please remove it or refactor all instances of it, Thank You

Reply:
@Mubangizi this has been fixed, kindly do check it out boss.